srcpkgs/net-tools/template

```diff
@@ -1,46 +1,45 @@
 # Template file for 'net-tools'
 pkgname=net-tools
-version=1.60.20181103git
+version=2.10
 revision=1
-_githash=0eebece8c964e3cfa8a018f42b2e7e751a7009a0
-wrksrc="${pkgname}-${_githash}"
+build_style="gnu-makefile"
+make_use_env=yes
 short_desc="Basic networking tools"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="0x5c <dev@0x5c.io>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/ecki/net-tools"
-distfiles="https://github.com/ecki/net-tools/archive/${_githash}.tar.gz"
-checksum=795cb4dd2618ed7ff9a48b9cb63b33780b47dfd7464c8e52ba734e6796cbab17
+distfiles="https://github.com/ecki/net-tools/archive/refs/tags/v${version}.tar.gz"
+checksum=8e186f65f0ff304989b3dc8b0067c3a07d36ae777da60667bdd6ccb2730908d2
 
 alternatives="
  ifconfig:ifconfig:/usr/bin/${pkgname}-ifconfig
  ifconfig:/usr/share/man/man1/ifconfig.1:/usr/share/man/man8/${pkgname}-ifconfig.8
  hostname:hostname:/usr/bin/${pkgname}-hostname
  hostname:/usr/share/man/man1/hostname.1:/usr/share/man/man8/${pkgname}-hostname.1"
 
-pre_build() {
-	vsed -i "s#/sbin#/bin#" Makefile
-	vsed -i "s#/usr##" man/Makefile
+post_patch() {
+	# Hacky and flaky, but upstream does not support $PREFIX !
+	vsed -i "s|/sbin|/bin|" Makefile
```

Comment:
This doesn't work without `-e` does it?

Reply:
I had a look at the manual right now and yeah, it shouldn't work at all. But I simply modified the vsed that was already there, and my changes to the vsed do reflect in the makefile.

I'll "fix" those two vsed, but now I'm curious what's going on